packages/client/src/rdf-functions.ts

```diff
@@ -1,13 +1,13 @@
-import * as RDFJS from 'rdf-js';
-import * as N3 from 'n3';
+import { DatasetCore } from 'rdf-js';
 import { Fetch } from './fetch';
+import { parseTurtle } from 'interop-utils';
```

Comment:
Move `parseTurtle` into rdf-utils as it is not specific to interop but an rdf operation.

Reply:
Currently all packages have `interop-` prefix, when we get to publishing we may need to create namespace for all of them. We only have some RDF helpers in `interop-utils`, are you ok with living it the way it is for now and later we can separate `interop-rdf-utils` when we see need for it? Otherwise I would need to rename `interop-utils` to `interop-rdf-utils` and we would create `interop-utils` when we need it.

Reply:
I see. Then it's fine as it is now.